app.py

```diff
@@ -95,6 +95,24 @@ def image():
 
     return 'OK'
 
+@app.route('/imageurl', methods=['POST'])
+@swag_from('swag/draw/imageurl.yml')
+def imageurl():
```

Comment:
Please have look at the new commit I just pushed. Works for me.

Reply:
Great stuff.
Really appreciate your input. :+1: 

However... :wink: 

I just ran your PR and noticed that the SwaggerUI now shows this:

![image](https://github.com/4ch1m/pixoo-rest/assets/3408832/44794249-1b9a-46b0-9625-55d6b1259467)

(The `Browse...` button completely disappeared; leaving no easy way to select a file.)

As compared to before:

![image](https://github.com/4ch1m/pixoo-rest/assets/3408832/a82f4992-80ec-422d-beac-cc4f5622479e)

I think this will be a regression for most users.

A dedicated `download` section (with new endpoints) probably would be a better solution overall; it also would maintain usability and keep things backwards-compatible.

Give me some time to evaluate this in the next few days.

Hope you understand and aren't too upset about my nitpicky decision. :smile: